src/pages/Bridge/RedeemForm/index.tsx

```diff
@@ -159,26 +196,29 @@ const RedeemForm = (): JSX.Element | null => {
           // set as a default on render.
           setHasPremiumRedeemVaults(true);
         }
-        if (vaultsWithRedeemableTokens.status === 'fulfilled') {
+        if (
+          vaultsWithRedeemableTokensResult.status === 'fulfilled' &&
+          vaultsWithRedeemableTokensResult.value.size > 0
```

Comment:
There was an edge case bug without this check I found in the process, so I added it although it was out of scope.

Reply:
The other day, I tested it. It can only be reproduced in an edge case where something goes wrong on the parachain side.